llvm/lib/Target/AMDGPU/GCNRegPressure.h

```diff
@@ -180,6 +190,13 @@ class GCNDownwardRPTracker : public GCNRPTracker {
 
   MachineBasicBlock::const_iterator getNext() const { return NextMI; }
 
+  // Return MaxPressure and clear it.
+  decltype(MaxPressure) moveMaxPressure() {
```

Comment:
Why use decltype? It's longer than GCNRegPressure.

Reply:
Done.